wordpress-mobile/WordPress-Android
# Detekt - Update 'LongParameterList' Configuration

This PR further configures [Detekt](https://github.com/detekt/detekt) to loosen the `LongParameterList` so that it accounts for dependency injection and ignores this rule when a constructor have parameters that are being injected, like on `ViewModel` or `UseCase`. As such, it makes is easier to work with the `LongParameterList` while not completely ignoring the rule.

PS: For more information read internal discussion here: `paqN3M-hh#comment-481`

To test:
- Pull the branch locally.
- Run `./gradlew WordPress:detekt` and make sure the task succeeds.

PR submission checklist:
